BarcodeSDK.MAUI.Example/ClassicUI/MVVM/ViewModels/BarcodeClassicComponentViewModel.cs

```diff
@@ -0,0 +1,58 @@
+using System.Windows.Input;
+using ScanbotSDK.MAUI.Barcode;
+using ScanbotSDK.MAUI.Barcode.Core;
+
+namespace ScanbotSDK.MAUI.Example.ClassicUI.MVVM.ViewModels;
+
+public class BarcodeClassicComponentViewModel : BaseViewModel
+{
+    public BarcodeClassicComponentViewModel()
+    {
+        BarcodeFormatConfigurations =
```

Comment:
How is this connected to the `BarcodeScannerView` ?
I tried updating the filter like below, but it still scans all barcodes, as I don't see it passing to the `BarocdeScannerView`.
```
new BarcodeFormatCommonConfiguration
{
    Formats = [BarcodeFormats.Code128]
}
```